pkg/kubelet/status/status_manager.go

```diff
@@ -154,6 +174,15 @@ func (m *manager) Start() {
 		return
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+		stateImpl, err := state.NewStateCheckpoint(m.stateFileDirectory, podStatusManagerStateFile)
```

Comment:
How does this compare to how we manage checkpoint state in pkg/kubelet/cm? Why do we need to pass a new directory parameter here?

Reply:
It is similar. 
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/cpu_manager.go#L211